src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -167,9 +182,21 @@ export class Inquiries extends React.Component<Props, State> {
     alert("Sorry, there was an error with your submission, please try again")
   }
 
+  trackSuccessfulSubmission() {
+    let ids: string[] = []
+    ids = this.state.selected_conversations.map(value => value.id)
```

Comment:
📇 🎉 

Besides in function parameters, I personally try to use as few explicit typings as possible and instead rely on the right code paths that will make TS implicitly type properly, like with the above `selectedConversations` variable.